### ISSUE TITLE
Return a distinct status for failure with no starttls support vs. failure b/c unsecured

### ISSUE THREAD

**Issue by vbrown608** [EFForg/starttls-scanner#51]
Right now both of these appear to return status code 2. It would be great if they could use different status numbers!

**Comment by vbrown608** [comment]
Different codes for queued and not present on the policy list would also be great!

**Comment by vbrown608** [comment]
It would be great if we could parse it like this:

https://github.com/EFForg/starttls-frontend/blob/master/js/scan.js#L76-L94

**Comment by sydneyli** [comment]
@vbrown608 It looks like that link no longer works?

**Comment by vbrown608** [comment]
Oops, moving target! See below.
```
function status_string(scan) {
  switch(scan.status) {
    case 0:
      switch (scan.extra_results.policylist.status) {
        case 0:
          return 'perfect'
        case 1:
          // Pending server using distinct status code for queued domain
          return 'pending'
        case 2:
          return 'not-submitted'
      }
    case 1:
      // Pending server using distinct status code for domain with starttls but bad configuration
      return 'fail-not-secured';
    case 2:
      return 'fail-no-support';
  }
}
```

**Comment by sydneyli** [comment]
This was closed by EFForg/starttls-check#13
